packages/create-react-app/createReactApp.js

```diff
@@ -216,6 +216,10 @@ function createApp(
   );
 
   const useYarn = useNpm ? false : shouldUseYarn();
+  const yarnUsesDefaultRegistry =
+    execSync('yarn config get registry').toString() ===
```

Comment:
Looking at `shouldUseYarn`, I can see it's using `yarnpkg` alias instead: https://github.com/facebook/create-react-app/blob/fb0b7758ff3869c99fa831c28e0ca71a08edfdd8/packages/create-react-app/createReactApp.js#L296

See https://github.com/facebook/create-react-app/pull/1365 for context.

Additionally, a user may not have Yarn installed, so it might be worth moving this check inside the `if (useYarn) {` check below?